src/nunit.gui.core/Model/Settings/GuiSettings.cs

```diff
@@ -81,9 +74,9 @@ public bool Maximized
         }
 
         private const string fontKey = "Font";
-        public Font Font
+        public ViewFont Font
```

Comment:
Since we can only store a string, why not just leave it string and have either the View or the platform-specific part of the Presenter translate it? Someone has to translate it anyway.